MIGRATION.md

```diff
@@ -0,0 +1,87 @@
+# Migrating to the `cloudant-python-sdk` library 
+This document is to assist in migrating from the legacy `python-cloudant` to the newly supported `cloudant-python-sdk`.
+
+## Initializing the client connection
+There are several ways to create a client connection in `cloudant-python-sdk`:
+1. [Enviornment variables](https://github.com/IBM/cloudant-python-sdk#authenticate-with-environment-variables)
+2. [External configuration file](https://github.com/IBM/cloudant-python-sdk#authenticate-with-external-configuration)
+3. [Programmatically](https://github.com/IBM/cloudant-python-sdk#authenticate-programmatically)
+
+[See the README](https://github.com/IBM/cloudant-python-sdk#1-retrieve-information-from-an-existing-database) for code examples on using enviornment variables.
+
+## Other differences
+1. The `cloudant-python-sdk` library does not support local dictionary caching of database and document objects.
+1. There's no context manager feature in `cloudant-python-sdk`.  You'll need to create the client, connect to the database, fetch and save a document as separate commands.
+1. Cloudant View, Search, and Query (aka `_find` endpoint) operations will contain raw JSON response content.  
+
+## Request mapping
+Here's a short list of which new functions to use for common operations:
+
+| `python-cloudant` operation           | `cloudant-python-sdk` API operation documentation link |
+|---------------------------------------|---------------------------------|
+|`Document('db_name', 'docid').fetch()` |[`getDocument`](https://cloud.ibm.com/apidocs/cloudant?code=python#getdocument)|
+|`db.get_view_result()`                 |[`postView`](https://cloud.ibm.com/apidocs/cloudant?code=python#postview)|
+|`db.get_query_result()`                |[`postFind`](https://cloud.ibm.com/apidocs/cloudant?code=python#postfind)|
+| `doc.exists()`                        |[`headDocument`](https://cloud.ibm.com/apidocs/cloudant?code=python#headdocument)|
+|`Document('db_name', 'docid').save()`  |[`putDocument`](https://cloud.ibm.com/apidocs/cloudant?code=python#putdocument)|
+
+
+[A more complete table](#reference-table) that links `cloudant-python-sdk` API operations with `python-cloudant` functions is provided at the end of this guide.
+
+The beta SDKs are generated from a more complete API spec and provide a significant number of operations that did not exist in the legacy libraries. See [the IBM Cloud API Documentation](https://cloud.ibm.com/apidocs/cloudant) to review request parameter and body options, code examples, and additional details for every endpoint.
+
+## Known Issues
+There's an [outline of known issues](https://github.com/IBM/cloudant-python-sdk/blob/master/KNOWN_ISSUES.md) in the `cloudant-python-sdk` repository.
+
+## Reference table
```

Comment:
Some description would be good, that describes what is a reference table would mean here.
Also a note would be good that highlight the parameter list might be differ at most of the functions.